executor/index_lookup_join_test.go

```diff
@@ -407,11 +486,14 @@ func (s *testSuite5) TestIssue27893(c *C) {
 	tk.MustQuery("select /*+ inl_hash_join(t2) */ count(*) from t1 join t2 on t1.a = t2.a").Check(testkit.Rows("1"))
 }
 
-func (s *testSuite5) TestPartitionTableIndexJoinAndIndexReader(c *C) {
+func TestPartitionTableIndexJoinAndIndexReader(t *testing.T) {
```

Comment:
please move to `index_lookup_join_serial_test.go`

Reply:
add parallel